lib/district.rb

```diff
@@ -1,41 +1,18 @@
-require_relative 'enrollment'
-require 'bigdecimal'
+require 'pry'
+
 class District
   attr_reader :name
+  attr_accessor :enrollment_data
 
   def initialize(name)
     @name = name[:name]
-
-  end
-
-  def enrollment
```

Comment:
Consistencies between variable names should also be cleaned up, so enrollment (get) and get_enrollment (misleading set) become :enrollment_data.